modules/java/CMakeLists.txt

```diff
@@ -51,12 +48,18 @@ bison_target(ModJavaGrammar
 
 find_package(Java 1.7)
 
-module_switch(ENABLE_JAVA "Enable java module" JAVA_FOUND Java_JAVAH_EXECUTABLE)
+if (JAVA_FOUND)
```

Comment:
This is probably a rebase error. The original `module_switch` should work.

Reply:
thanks. resolved